.github/workflows/set-version.yml

```diff
@@ -0,0 +1,16 @@
+name: Release and tag
+permissions:
+  contents: write
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+    types: [opened, edited] # we need this triggers for a title check
+
+jobs:
+  release-and-tag:
+    name: Call reusable set version
+    uses: NilFoundation/ci-cd/.github/workflows/reusable-set-version.yml@c6350a9f6339dcd6b4b73dd663c23f5d44177ff5
```

Comment:
Squash all the commits into one and put its hash here

Reply:
done